server/src/main.rs

```diff
@@ -85,18 +90,36 @@ impl LanguageServer for Backend {
     }
 
     async fn initialized(&self, _: InitializedParams) {
-        let workspace_uri = self.workspace_uri.lock().await;
+        // Update the sources and references directories
```

Comment:
It might be useful for the comment to mention that this is about loading configuration as this is the first time configuration is available.